examples/resources/hpegl_vmaas_load_balancer_monitor/nsx_t_lb_monitor.tf

```diff
@@ -1,7 +1,8 @@
 # (C) Copyright 2022 Hewlett Packard Enterprise Development LP
 
 resource "hpegl_vmaas_load_balancer_monitor" "tf_lb_monitor" {
-  name  =  "TEST-MONITOR"       
+  name  =  "TEST-MONITOR"  
+  lb_id = data.hpegl_vmaas_load_balancer.lb.id     
   description  = "Creating lb monitor"
   monitor_type = "LBTcpMonitorProfile"
```

Comment:
based on the monitor type, the payload changes in UI. Need to see how can this implemented better.